assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -160,6 +160,15 @@ export default {
           this.$('.topic-intro').prependTo(this.$('.main-link'))
           this.$('.topic-title').prependTo(this.$('.main-link'))
         }
+
+        if (socialMediaStyle) {
```

Comment:
That's what I did initially, but the TopicList component only has a "didInsertElement" event, and it doesn't get reinserted when switching between latest and new (or unread, or a specific category). I struggled for a while to have this run inside TopicList, but then abandoned that option for the inefficient TopicListItem.